### ISSUE TITLE
small error  in class_balanced_loss.py

### ISSUE THREAD

**Issue by eeric** [wgqtmac/cvprw2020#4]
on 87th line in class_balanced_loss.py: cb_loss = F.binary_cross_entropy_with_logits(input = logits,target = labels_one_hot, weights = weights)
there was small error that 'weights = weights' should be modified for 'weight = weights'.

**Comment by wgqtmac** [comment]
Thanks for your attention! We will change this error.
